ci-operator/step-registry/mirror-images/icsp-olm-operators/mirror-images-icsp-olm-operators-commands.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+mirror_auths="${SHARED_DIR}/mirror_auths"
+
+function set_proxy () {
+    if test -s "${SHARED_DIR}/proxy-conf.sh" ; then
+        echo "setting the proxy:"
+        cat "${SHARED_DIR}/proxy-conf.sh"
+        echo "source ${SHARED_DIR}/proxy-conf.sh"
+        source "${SHARED_DIR}/proxy-conf.sh"
+    else
+        echo "no proxy setting."
+    fi
+}
+
+# private mirror registry host
+# <public_dns>:<port>
+MIRROR_REGISTRY_HOST=`head -n 1 "${SHARED_DIR}/mirror_registry_url"`
+if [ ! -f "${SHARED_DIR}/mirror_registry_url" ]; then
+    echo "File ${SHARED_DIR}/mirror_registry_url does not exist."
+    exit 1
+fi
+echo "MIRROR_REGISTRY_HOST: $MIRROR_REGISTRY_HOST"
+
+# add the custom registry auth to the .dockerconfigjson of the cluster
+registry_cred=`head -n 1 "/var/run/vault/mirror-registry/registry_creds" | base64 -w 0`
+jq --argjson a "{\"${MIRROR_REGISTRY_HOST}\": {\"auth\": \"$registry_cred\"}}" '.auths |= . + $a' "/var/run/vault/mirror-registry/dockerconfigjson" > "${mirror_auths}"
+
+# Don't mirror OLM operators images, but create the ICSP for them.
+echo "=============>>> create ICSP for OLM operators"
+echo "$ oc adm catalog mirror -a ${mirror_auths} quay.io/openshift-qe-optional-operators/ocp4-index:latest ${MIRROR_REGISTRY_HOST} --manifests-only --to-manifests=/tmp/olm_mirror"
+if oc adm catalog mirror -a "${mirror_auths}" quay.io/openshift-qe-optional-operators/ocp4-index:latest ${MIRROR_REGISTRY_HOST} --manifests-only --to-manifests=/tmp/olm_mirror ; then
```

Comment:
Will we use proxy registries in Prow? If yes, it is better to use fixed ICSP file.

Reply:
```
apiVersion: operator.openshift.io/v1alpha1
kind: ImageContentSourcePolicy
metadata:
  name: image-policy-aosqe
spec:
  repositoryDigestMirrors:
  - mirrors:
    - MIRROR_REGISTRY_PLACEHOLDER1/openshifttest
    source: quay.io/openshifttest
  - mirrors:
    - MIRROR_REGISTRY_PLACEHOLDER2
    source: registry.redhat.io
  - mirrors:
    - MIRROR_REGISTRY_PLACEHOLDER2
    source: brew.registry.redhat.io
  - mirrors:
    - MIRROR_REGISTRY_PLACEHOLDER2
    source: registry.stage.redhat.io
  - mirrors:
    - MIRROR_REGISTRY_PLACEHOLDER1/openshift-qe-optional-operators
    source: quay.io/openshift-qe-optional-operators
  - mirrors:
    - MIRROR_REGISTRY_PLACEHOLDER2
    source: registry-proxy.engineering.redhat.com

```

Reply:
> Will we use proxy registries in Prow?

Yes. As we synced before, we won't mirror OLM operators' image, so we have to use the proxy registry.

Reply:
> it is better to use the fixed ICSP file.

Why? Any reasons? The current mapping:
```yaml
$ cat /tmp/olm_mirror/imageContentSourcePolicy.yaml
---
apiVersion: operator.openshift.io/v1alpha1
kind: ImageContentSourcePolicy
metadata:
  labels:
    operators.openshift.org/catalog: "true"
  name: ocp4-index-0
spec:
  repositoryDigestMirrors:
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/olmqe/sample-operator-bundle
    source: quay.io/olmqe/sample-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/amq7/amq-streams-bridge-rhel7
    source: registry.redhat.io/amq7/amq-streams-bridge-rhel7
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-ptp-operator
    source: registry.redhat.io/openshift4/ose-ptp-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/openshift-ose-elasticsearch-operator-bundle
    source: brew.registry.redhat.io/rh-osbs/openshift-ose-elasticsearch-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/openshift-ose-cluster-kube-descheduler-operator-bundle
    source: brew.registry.redhat.io/rh-osbs/openshift-ose-cluster-kube-descheduler-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/olmqe/sample-operator
    source: quay.io/olmqe/sample-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/amq7/amqstreams-rhel7-operator-metadata
    source: registry.stage.redhat.io/amq7/amqstreams-rhel7-operator-metadata
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/amq7/amq-streams-kafka-26-rhel7
    source: registry.redhat.io/amq7/amq-streams-kafka-26-rhel7
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/amq7/amq-streams-kafka-30-rhel8
    source: registry.redhat.io/amq7/amq-streams-kafka-30-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/openshift-ose-vertical-pod-autoscaler-operator-bundle
    source: brew.registry.redhat.io/rh-osbs/openshift-ose-vertical-pod-autoscaler-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/compliance/openshift-security-profiles-rhel8-operator
    source: registry.redhat.io/compliance/openshift-security-profiles-rhel8-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/amq7/amq-streams-kafka-27-rhel7
    source: registry.redhat.io/amq7/amq-streams-kafka-27-rhel7
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/compliance/openshift-selinuxd-rhel8
    source: registry.redhat.io/compliance/openshift-selinuxd-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift-logging/logging-curator5-rhel8
    source: registry.redhat.io/openshift-logging/logging-curator5-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/metallb-rhel8-operator
    source: registry.redhat.io/openshift4/metallb-rhel8-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-elasticsearch-proxy
    source: registry.redhat.io/openshift4/ose-elasticsearch-proxy
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/amq7/amq-streams-operator-bundle
    source: registry.stage.redhat.io/amq7/amq-streams-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-node-feature-discovery
    source: registry.redhat.io/openshift4/ose-node-feature-discovery
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift-update-service/openshift-update-service-rhel8
    source: registry.redhat.io/openshift-update-service/openshift-update-service-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift-logging/loki-rhel8-operator
    source: registry.redhat.io/openshift-logging/loki-rhel8-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-sriov-network-config-daemon
    source: registry.redhat.io/openshift4/ose-sriov-network-config-daemon
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-logging-fluentd
    source: registry.redhat.io/openshift4/ose-logging-fluentd
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift-logging/lokistack-gateway-rhel8
    source: registry.redhat.io/openshift-logging/lokistack-gateway-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/special-resource-rhel8-operator
    source: registry.redhat.io/openshift4/special-resource-rhel8-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift-update-service/openshift-update-service-rhel8-operator
    source: registry.redhat.io/openshift-update-service/openshift-update-service-rhel8-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/openshift-logging-elasticsearch-operator-bundle
    source: brew.registry.redhat.io/rh-osbs/openshift-logging-elasticsearch-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/openshift-ose-cluster-logging-operator-bundle
    source: brew.registry.redhat.io/rh-osbs/openshift-ose-cluster-logging-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-csi-external-provisioner
    source: registry.redhat.io/openshift4/ose-csi-external-provisioner
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/amq7/amq-streams-kafka-25-rhel7
    source: registry.redhat.io/amq7/amq-streams-kafka-25-rhel7
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/openshift4-performance-addon-operator-bundle-registry-container-rhel8
    source: brew.registry.redhat.io/rh-osbs/openshift4-performance-addon-operator-bundle-registry-container-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/external-dns-operator-bundle
    source: brew.registry.redhat.io/rh-osbs/external-dns-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/openshift-file-integrity-operator-bundle
    source: brew.registry.redhat.io/rh-osbs/openshift-file-integrity-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/amq7/amq-streams-rhel8-operator
    source: registry.redhat.io/amq7/amq-streams-rhel8-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-sriov-network-device-plugin
    source: registry.redhat.io/openshift4/ose-sriov-network-device-plugin
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/openshift-logging-loki-operator-bundle
    source: brew.registry.redhat.io/rh-osbs/openshift-logging-loki-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/metallb-rhel8
    source: registry.redhat.io/openshift4/metallb-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/compliance/openshift-compliance-content-rhel8
    source: registry.redhat.io/compliance/openshift-compliance-content-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift-update-service/cincinnati-operator-bundle
    source: registry.stage.redhat.io/openshift-update-service/cincinnati-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-ptp
    source: registry.redhat.io/openshift4/ose-ptp
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-sriov-network-operator
    source: registry.redhat.io/openshift4/ose-sriov-network-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-metering-hadoop
    source: registry.redhat.io/openshift4/ose-metering-hadoop
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-descheduler
    source: registry.redhat.io/openshift4/ose-descheduler
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/edo/external-dns-rhel8-operator
    source: registry.redhat.io/edo/external-dns-rhel8-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/amq7/amq-streams-kafka-27-rhel8
    source: registry.redhat.io/amq7/amq-streams-kafka-27-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/edo/external-dns-rhel8
    source: registry.redhat.io/edo/external-dns-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/amq7/amq-streams-bridge-rhel8
    source: registry.redhat.io/amq7/amq-streams-bridge-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/secondary-scheduler-operator-metadata-rhel-8
    source: brew.registry.redhat.io/rh-osbs/secondary-scheduler-operator-metadata-rhel-8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/amq7/amq-streams-rhel7-operator
    source: registry.redhat.io/amq7/amq-streams-rhel7-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/openshift-ose-metering-ansible-operator-bundle
    source: brew.registry.redhat.io/rh-osbs/openshift-ose-metering-ansible-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-sriov-dp-admission-controller
    source: registry.redhat.io/openshift4/ose-sriov-dp-admission-controller
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/openshift-ose-ptp-operator-bundle
    source: brew.registry.redhat.io/rh-osbs/openshift-ose-ptp-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-oauth-proxy
    source: registry.redhat.io/openshift4/ose-oauth-proxy
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift-logging/vector-rhel8
    source: registry.redhat.io/openshift-logging/vector-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift-logging/log-file-metric-exporter-rhel8
    source: registry.redhat.io/openshift-logging/log-file-metric-exporter-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/performance-addon-rhel8-operator
    source: registry.redhat.io/openshift4/performance-addon-rhel8-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/openshift-sandboxed-containers-operator-bundle
    source: brew.registry.redhat.io/rh-osbs/openshift-sandboxed-containers-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift-logging/fluentd-rhel8
    source: registry.redhat.io/openshift-logging/fluentd-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/albo/aws-load-balancer-rhel8-operator
    source: registry.redhat.io/albo/aws-load-balancer-rhel8-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/openshift-ose-metallb-operator-bundle
    source: brew.registry.redhat.io/rh-osbs/openshift-ose-metallb-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-elasticsearch-operator
    source: registry.redhat.io/openshift4/ose-elasticsearch-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/olmqe/learn-operator-bundle
    source: quay.io/olmqe/learn-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-sriov-cni
    source: registry.redhat.io/openshift4/ose-sriov-cni
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/openshift-ose-local-storage-operator-bundle
    source: brew.registry.redhat.io/rh-osbs/openshift-ose-local-storage-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-local-storage-static-provisioner
    source: registry.redhat.io/openshift4/ose-local-storage-static-provisioner
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-cluster-kube-descheduler-operator
    source: registry.redhat.io/openshift4/ose-cluster-kube-descheduler-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-logging-elasticsearch6
    source: registry.redhat.io/openshift4/ose-logging-elasticsearch6
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift-logging/elasticsearch-rhel8-operator
    source: registry.redhat.io/openshift-logging/elasticsearch-rhel8-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/kubebuilder/kube-rbac-proxy
    source: gcr.io/kubebuilder/kube-rbac-proxy
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-metering-presto
    source: registry.redhat.io/openshift4/ose-metering-presto
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/openshift-logging-cluster-logging-operator-bundle
    source: brew.registry.redhat.io/rh-osbs/openshift-logging-cluster-logging-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift-logging/opa-openshift-rhel8
    source: registry.redhat.io/openshift-logging/opa-openshift-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/compliance/openshift-compliance-openscap-rhel8
    source: registry.redhat.io/compliance/openshift-compliance-openscap-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-ghostunnel
    source: registry.redhat.io/openshift4/ose-ghostunnel
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/openshift-ose-cluster-nfd-operator-bundle
    source: brew.registry.redhat.io/rh-osbs/openshift-ose-cluster-nfd-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-local-storage-diskmaker
    source: registry.redhat.io/openshift4/ose-local-storage-diskmaker
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-cloud-event-proxy
    source: registry.redhat.io/openshift4/ose-cloud-event-proxy
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/openshift-ose-aws-efs-csi-driver-operator-bundle
    source: brew.registry.redhat.io/rh-osbs/openshift-ose-aws-efs-csi-driver-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift-logging/logging-loki-rhel8
    source: registry.redhat.io/openshift-logging/logging-loki-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-metering-reporting-operator
    source: registry.redhat.io/openshift4/ose-metering-reporting-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/openshift-ose-special-resource-operator-bundle
    source: brew.registry.redhat.io/rh-osbs/openshift-ose-special-resource-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/amq7/amq-streams-rhel7-operator-metadata
    source: registry.stage.redhat.io/amq7/amq-streams-rhel7-operator-metadata
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/amq7/amq-streams-kafka-24-rhel7
    source: registry.redhat.io/amq7/amq-streams-kafka-24-rhel7
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-csi-node-driver-registrar
    source: registry.redhat.io/openshift4/ose-csi-node-driver-registrar
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-sriov-operator-must-gather
    source: registry.redhat.io/openshift4/ose-sriov-operator-must-gather
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/compliance/openshift-compliance-rhel8-operator
    source: registry.redhat.io/compliance/openshift-compliance-rhel8-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift-logging/elasticsearch-proxy-rhel8
    source: registry.redhat.io/openshift-logging/elasticsearch-proxy-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-cluster-nfd-operator
    source: registry.redhat.io/openshift4/ose-cluster-nfd-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/openshift-security-profiles-operator-bundle
    source: brew.registry.redhat.io/rh-osbs/openshift-security-profiles-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-aws-efs-csi-driver-rhel8-operator
    source: registry.redhat.io/openshift4/ose-aws-efs-csi-driver-rhel8-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-local-storage-operator
    source: registry.redhat.io/openshift4/ose-local-storage-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/compliance/openshift-file-integrity-rhel8-operator
    source: registry.redhat.io/compliance/openshift-file-integrity-rhel8-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-cluster-logging-operator
    source: registry.redhat.io/openshift4/ose-cluster-logging-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-sriov-infiniband-cni
    source: registry.redhat.io/openshift4/ose-sriov-infiniband-cni
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-csi-livenessprobe
    source: registry.redhat.io/openshift4/ose-csi-livenessprobe
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/albo/aws-load-balancer-controller-rhel8
    source: registry.redhat.io/albo/aws-load-balancer-controller-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/amq7/amq-streams-kafka-23-rhel7
    source: registry.redhat.io/amq7/amq-streams-kafka-23-rhel7
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/openshift-ose-sriov-network-operator-bundle
    source: brew.registry.redhat.io/rh-osbs/openshift-ose-sriov-network-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/amq7/amq-streams-kafka-31-rhel8
    source: registry.redhat.io/amq7/amq-streams-kafka-31-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/olmqe/learn-operator
    source: quay.io/olmqe/learn-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/amq7/amq-streams-kafka-28-rhel8
    source: registry.redhat.io/amq7/amq-streams-kafka-28-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-kube-rbac-proxy
    source: registry.redhat.io/openshift4/ose-kube-rbac-proxy
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-metering-hive
    source: registry.redhat.io/openshift4/ose-metering-hive
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift-secondary-scheduler-operator/secondary-scheduler-operator-rhel8
    source: registry.redhat.io/openshift-secondary-scheduler-operator/secondary-scheduler-operator-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-aws-efs-csi-driver-container-rhel8
    source: registry.redhat.io/openshift4/ose-aws-efs-csi-driver-container-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift-logging/kibana6-rhel8
    source: registry.redhat.io/openshift-logging/kibana6-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/ubi8/openjdk-11
    source: registry.access.redhat.com/ubi8/openjdk-11
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-sriov-network-webhook
    source: registry.redhat.io/openshift4/ose-sriov-network-webhook
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/openshift-compliance-operator-bundle
    source: brew.registry.redhat.io/rh-osbs/openshift-compliance-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/frr-rhel8
    source: registry.redhat.io/openshift4/frr-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift-logging/cluster-logging-rhel8-operator
    source: registry.redhat.io/openshift-logging/cluster-logging-rhel8-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-logging-kibana6
    source: registry.redhat.io/openshift4/ose-logging-kibana6
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift-sandboxed-containers/osc-rhel8-operator
    source: registry.redhat.io/openshift-sandboxed-containers/osc-rhel8-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-logging-curator5
    source: registry.redhat.io/openshift4/ose-logging-curator5
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift-logging/elasticsearch6-rhel8
    source: registry.redhat.io/openshift-logging/elasticsearch6-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-vertical-pod-autoscaler-rhel8
    source: registry.redhat.io/openshift4/ose-vertical-pod-autoscaler-rhel8
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/rh-osbs/aws-load-balancer-operator-bundle
    source: brew.registry.redhat.io/rh-osbs/aws-load-balancer-operator-bundle
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-metering-ansible-operator
    source: registry.redhat.io/openshift4/ose-metering-ansible-operator
  - mirrors:
    - ec2-18-254-18-184.us-gov-east-1.compute.amazonaws.com:5000/openshift4/ose-vertical-pod-autoscaler-rhel8-operator
    source: registry.redhat.io/openshift4/ose-vertical-pod-autoscaler-rhel8-operator
```